### DISCUSSION TITLE
MUST Restart firefox after launching shadowsocks

### DISCUSSION THREAD

**Discussion by lsq93325** [shadowsocks/shadowsocks-windows#3072] [Q&A]
description

I use Firefox to explore the Internet, and for somereason I have to use shadowsocks. However, if I open Firefox before launching shadowsocks, ss will not forward my packet to remote server. In this case, I have to close all the tabs to exit Firefox entirely and then can I normally use shadowsock. This is not convenient. because I want to keep the opend web pages. Can someone help out?

**Comment by chenshaoju** [comment]
This is the Firefox designed like this.

In my case, I set Shadowsocks auto startup when the computer tunes on.

**Reply by lsq93325** [reply]
Thanks. However, I find that Firefox can seamlessly using shadowsocks without restarting on MacOS. However, on Windows  one have to restart Firefox.
